programs/storage_api/src/storage_contract.rs

```diff
@@ -456,7 +475,7 @@ mod tests {
 
         account.account.data.resize(4 * 1024, 0);
```

Comment:
Maybe I missed this. Use the size const here?

Reply:
✔️